### ISSUE TITLE
loss is nan

### ISSUE THREAD

**Issue by w5688414** [Thinklab-SJTU/R3Det_Tensorflow#17]
I use R3Det to train my own images, But I found the loss is nan

> 2020-03-30 13:01:15: global_step:59360  current_step:59360
per_cost_time:0.968s
cls_loss:0.912
reg_loss:nan
refine_cls_loss:0.000
refine_reg_loss:0.000
refine_cls_loss_stage3:0.000
refine_reg_loss_stage3:0.000
total_losses:nan
mask_loss:0.000

the default learning rate is 5e-4, very low

**Comment by jyshee** [comment]
got the same problem

**Comment by adamzhu1990** [comment]
the same problem with me, have you ever solve it ? maybe it comes from the rbbox_transform function in lib/box_utils/bbox_transform.py

**Comment by zsh1234567** [comment]
First of all, thanks to the author for the code.
I have used the parameters given by the author, the results I got are as follows.
However,"I find the refine_reg_loss_stage3:0.00",Its value is 0, is this normal?

2020-04-19 20:51:29: global_step:2560 current_step:2560
per_cost_time:1.864s
cls_loss:0.979
reg_loss:1.603
refine_cls_loss:1.234
refine_reg_loss:1.127
refine_cls_loss_stage3:0.001
refine_reg_loss_stage3:0.000
total_losses:4.946

**Comment by yangxue0827** [comment]
The reason for this (NAN after 'bbox_transform.py:98: RuntimeWarning: divide by zero encountered in log') is that the model predicts a very small box (w and h are close to 0), which leads to a problem with the encoding and decoding of the bounding box. I will upload the repaired code and it is still being debugged. @w5688414 @zsh1234567 @jyshee
